authors/apps/authentication/views.py

```diff
@@ -62,11 +63,7 @@ def post(self, request):
             'title': subject,
             'username': username,
         })
-        body_content = strip_tags(message)
-        msg = EmailMultiAlternatives(subject, body_content, sender, to=[email])
-        msg.attach_alternative(message, "text/html")
-        msg.send()
-
+        base_mail_send(subject, sender, email, message)
```

Comment:
Hey @Nicanor008 , good use of the base_mail_send() function to send all emails in our application. Keep up on **Good Code Refactor**